docs/streams/developer-guide/dsl-api.html

```diff
@@ -764,10 +764,10 @@ <h4 class="anchor-heading"><a id="streams_concepts_globalktable" class="anchor-l
                         <td>
                             <p class="first">Manually trigger repartitioning of the stream with desired number of partitions. (<a class="reference external" href="/{{version}}/javadoc/org/apache/kafka/streams/kstream/KStream.html#repartition--">details</a>)</p>
 
-                            <code><span class="pre">repartition()</span></code> is similar to <code><span class="pre">through()</span></code> however Kafka Streams will manage the topic for you.
+                            Kafka Streams will manage the topic for <code><span class="pre">repartition()</span></code>.
                             Generated topic is treated as internal topic, as a result data will be purged automatically as any other internal repartition topic.
                             In addition, you can specify the desired number of partitions, which allows to easily scale in/out downstream sub-topologies.
-                            <code><span class="pre">repartition()</span></code> operation always triggers repartitioning of the stream, as a result it can be used with embedded Processor API methods (like <code><span class="pre">transform()</span></code> et al.) that do not trigger auto repartitioning when key changing operation is performed beforehand.
```

Comment:
Instead of removing the reference to `transform()` we should update it to `process()` with replaces `transform()`.

Reply:
Thanks for the review. Updated it.

<img width="972" alt="Screenshot 2025-04-08 at 1 47 36 PM" src="https://github.com/user-attachments/assets/9a7c7ac5-f2c3-4ab9-9f5d-d5ff063ea6ad" />